Automattic/sensei
# Add last onboarding step and remove/deprecate some code

Part of #5647

### Changes proposed in this Pull Request

* It adds the last step to the new onboarding, which installs the plugins based on what the user selected in the previous steps. It doesn't include the installation in this PR yet.
* It removes some Setup Wizard steps that we won't have anymore.
* It also deprecates some code related to WCCOM extension installation, since we won't use it anymore.

### Testing instructions


* Navigate through the setup wizard until the last step, and make sure it works properly.


### Deprecated Code

* `Sensei_Extensions::get_installed_plugins`
* `Sensei_Setup_Wizard::close_wccom_install`
* `Sensei_Usage_Tracking::log_wccom_plugin_install`
* `Sensei_Utils::get_woocommerce_connect_data`
